cli/util/fs.rs

```diff
@@ -492,6 +492,68 @@ pub async fn remove_dir_all_if_exists(path: &Path) -> std::io::Result<()> {
   }
 }
 
+mod clone_dir_imp {
+  use deno_core::error::AnyError;
+
+  use super::copy_dir_recursive;
+  use std::path::Path;
+
+  cfg_if::cfg_if! {
```

Comment:
Nit (this isn't that bad without `cfg_if`):
```suggestion
#[cfg(target_vendor = "apple")]
mod apple {
  // ...
}

#[cfg(target_vendor = "apple")]
pub(super) fn clone_dir_recursive(from: &Path, to: &Path) -> Result<(), AnyError> {
  apple::clone_dir_resursive(from, to)
}

#[cfg(not(target_vendor = "apple"))]
pub(super) fn clone_dir_recursive(from: &Path, to: &Path) -> Result<(), AnyError> {
  // ...
}
```